examples/experimental/dagster-blueprints/dagster_blueprints/cli.py

```diff
@@ -0,0 +1,175 @@
+import json
+import os
+import shutil
+import subprocess
+import zipfile
+from importlib import import_module
+from pathlib import Path
+from typing import List, Optional, Sequence
+
+import click
+from dagster import _check as check
+from dagster._cli.workspace.cli_target import has_pyproject_dagster_block
+from dagster._core.definitions.load_assets_from_modules import find_objects_in_module_of_types
+from dagster._core.remote_representation.origin import ManagedGrpcPythonEnvCodeLocationOrigin
+from dagster._core.workspace.load_target import PyProjectFileTarget
+from dagster._utils.warnings import disable_dagster_warnings
+
+from dagster_blueprints.load_from_yaml import YamlBlueprintsLoader
+
+from .version import __version__
+
+
+def infer_vscode_path(base_path: Path) -> Optional[str]:
```

Comment:
Right now we store the built extension in the `.vscode` directory - maybe we should put it somewhere else?